common/src/main/java/apoc/util/JsonUtil.java

```diff
@@ -123,7 +125,7 @@ public static Stream<Object> loadJson(
                     ? stream
                     : stream.map((value) ->
                             JsonPath.parse(value, getJsonPathConfig(options)).read(path));
-        } catch (IOException e) {
+        } catch (IOException | URISyntaxException | URLAccessValidationError e) {
             if (!failOnError) {
                 return Stream.of();
             } else {
```

Comment:
Should the URI/URL exceptions also fallback on an empty stream or always throw an exception?

Reply:
Nah, throw an exception. Then all these places you've added these two, we may as well just throw Exception instead of specifying 3.